builtin/logical/ssh/backend_test.go

```diff
@@ -668,29 +680,33 @@ func TestSSHBackend_CredsForZeroAddressRoles_dynamic(t *testing.T) {
 
 func TestSSHBackend_CA(t *testing.T) {
 	testCases := []struct {
-		name        string
-		tag         string
-		algoSigner  string
-		expectError bool
+		name         string
+		tag          string
+		caPublicKey  string
+		caPrivateKey string
+		algoSigner   string
+		expectError  bool
 	}{
-		{"defaultSignerSSHDSupport", dockerImageTagSupportsRSA, "", false},
-		{"rsaSignerSSHDSupport", dockerImageTagSupportsRSA, ssh.SigAlgoRSA, false},
-		{"rsa2SignerSSHDSupport", dockerImageTagSupportsRSA, ssh.SigAlgoRSASHA2256, false},
-		{"rsa2SignerNoSSHDSupport", dockerImageTagSupportsNoRSA, ssh.SigAlgoRSASHA2256, false},
-		{"defaultSignerNoSSHDSupport", dockerImageTagSupportsNoRSA, "", true},
+		{"RSAKey/EmptyAlgoSigner/ImageSupportsRSA1", dockerImageTagSupportsRSA1, testCAPublicKey, testCAPrivateKey, "", false},
```

Comment:
Think you might want to go multi-line here for readability.

Reply:
Also, mind replacing these slashes with dashes or something?  It makes them look like sub-sub-sub tests, and might complicate future efforts to list tests and schedule them across CI executors.

Reply:
Yep, I'll address both suggestions.